modin/core/execution/ray/common/utils.py

```diff
@@ -196,3 +197,54 @@ def initialize_ray(
         NPartitions._put(num_gpus)
     else:
         NPartitions._put(num_cpus)
+
+
+@ray.remote
+class SignalActor:  # pragma: no cover
```

Comment:
I would prefer this was in task_wrapper.py or class_wrapper.py file. Maybe, some generic file should be created to put content from task_wrapper.py and `SignalActor` there.

Reply:
https://github.com/modin-project/modin/issues/3861